php/class-post-type.php

```diff
@@ -675,4 +711,68 @@ public function hide_disabled_publishing_actions( $post ) {
 		</style>
 		<?php
 	}
+
+	/**
+	 * Get conflicts settings
+	 *
+	 * @param \WP_Post $post post to compare conflict values.
+	 * @param array    $settings setting to search for optional.
+	 *
+	 * @return array
+	 */
+	public function get_conflicts_setting( $post, $settings = array() ) {
+		global $wpdb;
+		if ( $post ) {
```

Comment:
Either the post type for `$post` should be checked to be `customize_snapshot` or the a `Customize_Snapshot` could be passed as the argument instead.